js/private/test/image/structure/digests.sum

```diff
@@ -1,2 +1,2 @@
-099acc2677cf8fbcd5c49d79169a21a8f541234d312e8e71b260af484093e061
+db4b47b2410911b723bed458f4be3ac6eae5859c232e924f7401856224233196
```

Comment:
??? how'd this get updated?

Reply:
I'm not sure. Still trying to track this down..

Reply:
My best guess is that somehow rules_js files are being included in the shasum, perhaps defined by the files and dirs here? https://github.com/aspect-build/rules_js/blob/main/js/private/test/image/structure/app_structure.mf

Reply:
@thesayyn Do you know what's causing this?

Reply:
Oh actually I'm getting the same error on main, so I think it's unrelated.

Reply:
Removed the changes.

Reply:
Fixed in https://github.com/aspect-build/rules_js/pull/1012.